com.github.yucefsourani.pysavetube.json

```diff
@@ -0,0 +1,115 @@
+{
+    "app-id" : "com.github.yucefsourani.pysavetube",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "pysavetube.py",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri",
+        "--socket=pulseaudio"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "intltool",
```

Comment:
There is manifest for this in `shared-modules`: https://github.com/flathub/shared-modules/

It's better to use that. (albeit not a blocker)

Reply:
thanks